src/components/views/settings/tabs/LabsSettingsTab.js

```diff
@@ -53,13 +53,6 @@ export class LabsSettingToggle extends React.Component {
     }
 
     _onChange = async (checked) => {
-        if (this.props.featureId === "feature_lazyloading") {
-            const confirmed = await this._onLazyLoadChanging(checked);
```

Comment:
if we're dropping the homeserver check, please also delete the function above.

Reply:
good catch